tests/graphql/passed/test-resolve-with-vars.graphql

```diff
@@ -1,5 +1,5 @@
 #> server-sdl
-schema @server(vars: {id: "1"}) {
+schema @server(vars: [{key: "id",value: "1"}]) {
```

Comment:
I wonder why to change the public api in how the vars are defined?

Reply:
We couldn't do this :

```graphql
@server(vars: {this-is-id: "1"})
```

But now we can:

```graphql
@server(vars: [{key: "this-is-id", value: "1"}])
```

Reply:
makes sense

Reply:
this is needed to support Intellisense in the GraphQL SDL;